css/fieldmanager.css

```diff
@@ -274,26 +453,26 @@ div.fm-group-label-wrapper .fmjs-remove {
 
 /* Grid editor CSS */
 .grid-toggle-wrapper {
-	padding: 7px;
-	margin: 5px 5px 5px 0;
 	overflow: scroll;
 	max-height: 500px;
+	margin: 5px 5px 5px 0;
+	padding: 7px;
 }
 
 .grid-toggle-wrapper.with-grid {
 	padding: 5px;
-	background: #fff;
 	border: solid 2px #ccc;
+	background: #fff;
 }
 
 .grid-activate {
```

Comment:
Seems like it's unrelated to these new styles, but the input styles for grid aren't working well. If you try to input data into a grid (at least in Chrome) you'll see what I mean`.handsontableInputHolder` needs to have a width set.